EmberFlexberry/Objects/SuggestionType.cs

```diff
@@ -60,8 +61,9 @@ namespace EmberFlexberryDummy
             "Name as \'Name\'",
             "Moderated as \'Moderated\'",
             "Parent as \'Parent\'",
-            "Parent.Name as \'Name\'"}, Hidden=new string[] {
-            "Parent.Name"})]
+            "Parent.Name as \'Name\'"}, Hidden = new string[] {
```

Comment:
Вроде свойство не только в скрытые должно генерироваться.
```suggestion
            "Parent.Name as \'Name\'",
            "IsParentRecord"}, Hidden = new string[] {
```